pkg/metrics/tls.go

```diff
@@ -170,15 +173,24 @@ func (t *tlsConfigurerImpl) updateClientCA(cm *v1.ConfigMap) {
 		return
 	}
 	if caFile, ok := cm.Data[clientCAKey]; ok {
-		certBlock, _ := pem.Decode([]byte(caFile))
-		cert, err := x509.ParseCertificate(certBlock.Bytes)
-		if err != nil {
-			log.Errorw("Unable to parse client CA", zap.Error(err))
-			return
+		log.Infof("Updating secure metrics client CAs based on %s/%s", t.clientCANamespace, t.clientCAConfigMap)
+		var clientCAs []*x509.Certificate
+		for block, rest := pem.Decode([]byte(caFile)); block != nil; block, rest = pem.Decode(rest) {
```

Comment:
Let's use `cfssl/helpers.ParseCertificatesPEM` instead of writing our own function to parse multiple PEM encoded certs.